config/sync/block.block.webtools_etrans.yml

```diff
@@ -0,0 +1,23 @@
+uuid: 9ba4d46f-3c31-430d-8207-990dfaf4d4ba
+langcode: en
+status: true
+dependencies:
+  module:
+    - eif
+    - node
+    - oe_webtools_etrans
+  theme:
+    - joinup_theme
+id: webtools_etrans
+theme: joinup_theme
+region: content
+weight: -25
+provider: null
+plugin: oe_webtools_etrans
+settings:
+  id: oe_webtools_etrans
+  label: 'OpenEuropa Webtools ETrans'
+  provider: oe_webtools_etrans
+  label_display: '0'
+  render_as: link
```

Comment:
In the ticket's wireframes I've noticed that they are requiring a button rather than a link. I had a discussion with Filip and he confirmed.

Reply:
I suppose the reason for this is to make sure that search engines don't follow it? I gathered from the mockups that this was intended to be a link styled as a button. I can change it.

Reply:
This was discussed with the business analysts. They also opt for the button. I will change it and update tests.